### PR TITLE
Lesson 5

### PR DESCRIPTION
**Pull request by LizavetBa**
finished fifth HT

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,17 +1,59 @@
...
+    <div class="box laptop">
+        <img src="images/laptop.png" alt="LAPTOP">
+        <p> Book online</p>
+    </div>
+    <div class="box ok">
+        <img src="images/ok.png" alt="REVIEWS">
+        <p> Real guest reviews</p>
+    </div>
+</div>
+<div class="Blue">
```
Comment (almuly) [inline index.html]: in this case for items in columns better to use ul and li tags

Comment (almuly) [inline index.html]: bad neme for class, it doesn't described for what you use it. better .background_wrapper
and never use capital letter in nemes of classes

File: styles/style.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    font-size: 24px;
+    text-align: center;
+    padding-top: 8px;
+}
+
+.Blue {
+    color: white;
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 1440px;
```
Comment (almuly) [inline styles/style.css]: for align conten in the center of block use margin: 0 auto

File: styles/style.css [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    padding: 120px 0 200px 104px;
```
Comment (almuly) [inline styles/style.css:109]: after aligning you need to check this paddings

File: styles/style.css [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+    background-color: #10367c;
```
Comment (almuly) [inline styles/style.css:110]: according to design your background must  cover all scrren, so for that you can create new div with class and transfer backgrounfd color there

File: styles/style.css [hunk 5]
```diff
@@ -0,0 +1,123 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
+
+* {
+    font-family: 'Roboto', sans-serif;
+    max-width: 1440px;
+    margin: 0 auto;
+}
+
+h1 {
+    display: flex;
```
Comment (almuly) [inline styles/style.css:10]: Wrong font-weight

File: styles/style.css [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+    justify-content: center;
+    width: 306px;
+    height: 47px;
```
Comment (almuly) [inline styles/style.css]: don't use fix height, better to add padding between title and line

File: styles/style.css [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+    margin: 120px 567px 0 567px;
```
Comment (almuly) [inline styles/style.css:14]: Unneccessary, you can yse paading top and thats all

File: styles/style.css [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+    margin-top: 60px;
+    margin-bottom: 100px;
+}
+
+.box {
+    width: 296px;
+    height: 176px;
+}
+
+.headphones img {
```
Comment (almuly) [inline styles/style.css:39]: add img wrapper for all images wuth property width:100px, height, 100px, then for images you can use property height auto and you didn't  need to describe five different classes

**Review by almuly — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (LizavetBa) [reply index.html]: added lists

Comment (LizavetBa) [reply index.html]: renamed

File: styles/style.css (hunk 2 above)
Comment (LizavetBa) [reply styles/style.css]: added margin

File: styles/style.css (hunk 3 above)
Comment (LizavetBa) [reply styles/style.css:109]: correct paddings

File: styles/style.css (hunk 5 above)
Comment (LizavetBa) [reply styles/style.css:10]: added font-weight 500

File: styles/style.css (hunk 6 above)
Comment (LizavetBa) [reply styles/style.css]: removed height and added padding

File: styles/style.css (hunk 8 above)
Comment (LizavetBa) [reply styles/style.css:39]: but every img has its own size and in Figma there diffirent sizes for each img

File: styles/style.css (hunk 7 above)
Comment (LizavetBa) [reply styles/style.css:14]: done

**Comment by LizavetBa** [comment]
fixed

**Pull request closed without merging**